src/Sulu/Component/Rest/FlattenExceptionNormalizer.php

```diff
@@ -42,6 +42,13 @@ public function __construct(
         $this->translator = $translator;
     }
 
+    public function getSupportedTypes(?string $format): array
+    {
+        return [
+            FlattenException::class => false,
```

Comment:
Thx for the pull request. Can you link the documentation in the Pull Request description. I'm not sure what the boolean flag does here.

Reply:
I have added a link. In short the flag will be used as an indicator for the cachability of the result of the supports* method. It will replace the `CacheableSupportsMethodInterface::hasCacheableSupportsMethod`  in Symfony 7.

Reply:
The reason why I used `false` for the `FlattenException` is that the support depends on the value of `$context['messenger_serialization']` ;-)